api/v1/helpers/verbs/doGetHierarchy.js

```diff
@@ -29,6 +29,32 @@ function doGetHierarchy(resultObj) {
   const params = resultObj.params;
   const depth = Number(params.depth.value);
   resultObj.dbStartTime = Date.now();
+
+  const fields = params.fields.value;
+  const filterFields = fields && fields.length;
+  const subjectTags = params.subjectTags.value;
+  const filterByTags = subjectTags && subjectTags.length;
```

Comment:
yes, apply to top-level object object

(we could always add additional stuff like "sampleFields=status,previousStatus..." later)